scripts/cmapi-start

```diff
@@ -22,9 +23,4 @@ echo CMAPI PID = $!
 # Start MariaDB
 /usr/share/mysql/mysql.server start
 
-# Start Columnstore Engine Proceses For Single Node
-if [[ $(hostname -s) == $PM1 ]]; then
-  curl -s -X PUT "https://$PM1:8640/cmapi/0.4.0/cluster/add-node" --header 'Content-Type:application/json' --header "x-api-key:$API_KEY" --data '{"timeout":60, "node": "'$PM1'"}' -k
-fi
-
```

Comment:
Observed some weirdness adding node automatically. Sometimes my machine would have one node, sometimes it didn't. Might have just been me. Feel free to ignore